webui/react/src/components/Link.tsx

```diff
@@ -1,43 +1,46 @@
 import React, { MouseEvent, MouseEventHandler, PropsWithChildren, useCallback } from 'react';
 
-import { handlePath, windowOpenFeatures } from 'utils/routes';
+import { handlePath, serverAddress, windowOpenFeatures } from 'utils/routes';
 
 import css from './Link.module.scss';
 
-interface Props {
+export interface LinkProps {
   className?: string;
   disabled?: boolean;
   inherit?: boolean;
   isButton?: boolean;
   path?: string;
   popout?: boolean;
+  proxy?: boolean;
   onClick?: MouseEventHandler;
 }
 
-const Link: React.FC<Props> = ({
+const Link: React.FC<LinkProps> = ({
   path = '#',
   popout,
+  proxy,
   onClick,
   ...props
-}: PropsWithChildren<Props>) => {
+}: PropsWithChildren<LinkProps>) => {
   const classes = [ css.base ];
   const rel = windowOpenFeatures.join(' ');
+  const proxyPath = `${proxy ? serverAddress(true) : ''}${path}`;
```

Comment:
hmm how about naming the prop `noProxy` IMO that could be simpler? we're whitelisting some links to avoid proxy or to connect "direct"ly.